devel/external/AEGIS/AEGIS_25.2_training_sample_readme.txt

```diff
@@ -0,0 +1,109 @@
+This README file describes the AEGIS F814W<25.2 training sample, which is a
+randomly ordered sample of galaxies in the AEGIS field with AB magnitude<25.2, in F606W(V) and F814W(I) bands.
```

Comment:
The flux limit is imposed in both bands?  I thought this was something we had discussed as a possible source of unexpected selection effects (because it omits certain regions of color space once you're near the flux limit in one band).  And more to the point, imposing selection in both bands means that you can't make catalogs that correspond to each other.  It seems that for many use cases, we will want a cut in one band, with images provided in both bands regardless of the flux in the other band (e.g., so dropouts will not be excluded from the catalog).